docs/source/models/causes/alzheimers/preclinical_and_mci_gbd_2021/index.rst

```diff
@@ -0,0 +1,179 @@
+..
+  Section title decorators for this document:
+
+  ==============
+  Document Title
+  ==============
+
+  Section Level 1 (#.0)
+  +++++++++++++++++++++
+
+  Section Level 2 (#.#)
+  ---------------------
+
+  Section Level 3 (#.#.#)
+  ~~~~~~~~~~~~~~~~~~~~~~~
+
+  Section Level 4
+  ^^^^^^^^^^^^^^^
+
+  Section Level 5
+  '''''''''''''''
+
+  The depth of each section level is determined by the order in which each
+  decorator is encountered below. If you need an even deeper section level, just
+  choose a new decorator symbol from the list here:
+  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+  And then add it to the list of decorators above.
+
+.. _2021_cause_alzheimers_preclinical_mci:
+
+===============================================================
+Alzheimer's disease  with preclinical and MCI stages (GBD 2021)
+===============================================================
+
+.. contents::
+  :local:
+
+.. list-table:: Abbreviations
+  :header-rows: 1
+
+  * - Abbreviation
+    - Definition
+  * - AD
+    - Alzheimer's Disease
+  * - BBBM
+    - Blood-Based Biomarker
+  * - CSU
+    - Client Services Unit
+  * - MCI
+    - Mild Cognitive Impairment
+  * - YLD
+    - Years Lived with Disability
+  * - YLL
+    - Years of Life Lost
+
+Disease Overview
+++++++++++++++++
+
+GBD 2021 Modeling Strategy
+++++++++++++++++++++++++++
+
+Restrictions
+------------
+
+The following table describes any restrictions in GBD 2021 on the
+effects of the cause "Alzheimer's disease and other dementias" (such as
+being only fatal or only nonfatal), as well as restrictions on the ages
+and sexes to which the cause applies.
+
+.. list-table:: GBD 2021 Cause Restrictions
+  :widths: 15 15 20
+  :header-rows: 1
+
+  * - Restriction Type
+    - Value
+    - Notes
+  * - Male only
+    - False
+    -
+  * - Female only
+    - False
+    -
+  * - YLL only
+    - False
+    -
+  * - YLD only
+    - False
+    -
+  * - YLL age group start
+    - 40 to 44
+    - age_group_id = 13
+  * - YLL age group end
+    - 95 plus
+    - age_group_id = 235
+  * - YLD age group start
+    - 40 to 44
```

Comment:
If MCI has a disability weight and starts before GBD's dementia cause, will we have YLDs in younger age groups?

Reply:
Good point - these restrictions apply to the AD state, but MCI doesn't have any restrictions (unless we want to impose some). I'll try to clarify.